src/Platform/Microsoft.Testing.Extensions.CrashDump/CrashDumpEnvironmentVariableProvider.cs

```diff
@@ -132,11 +132,10 @@ public Task UpdateAsync(IEnvironmentVariables environmentVariables)
 
     public Task<ValidationResult> ValidateTestHostEnvironmentVariablesAsync(IReadOnlyEnvironmentVariables environmentVariables)
     {
-        StringBuilder errors = new();
```

Comment:
Interesting. I would have expected a warning for unused local variable on the netstandard2.0 build.